README.md

```diff
@@ -11,9 +11,23 @@ Ansible playbook to setup a **SMTP** server, with:
 ```bash
 ansible-playbook -i inventory --user=root --become setup-mailserver.yml
 ```
+or
+
+```bash
+sudo ansible-playbook -i inventory --user=root --become setup-mailserver.yml --private-key=<your_private_key_path>
+```
 
 ## Test sending email
 ```bash
 sudo apt-get install mailutils -y   # For Debian/Ubuntu
 echo "This is a test email" | mail -s "Test Email" -r test@qa-smtp.cyverse.at mojib.wali@tugraz.at
 ```
+
+## Test sending email from external services
+1. create a new user in the mail server
```

Comment:
Ansible should create atleast one user that could send emails.
try adding a ansible task/role after the server is configured to create a test user - and the user information should be added to /defaults so it is possible to override while running the playbook.